tasmota/tasmota_xdrv_driver/xdrv_49_pid.ino

```diff
@@ -276,12 +292,14 @@ void PIDShowSensor() {
     JsonParserToken value_token = root[PID_LOCAL_SENSOR_NAME].getObject()[PSTR(PID_LOCAL_SENSOR_TYPE)];
     if (value_token.isNum()) {
       sensor_reading = value_token.getFloat();
+      //AddLog(LOG_LEVEL_DEBUG_MORE, PSTR("PID: PIDProcessSensor: Got isNum: %s"), buf.c_str());
```

Comment:
Remove commented out code.

Reply:
No need to. Always usefull when debugging as as commented take up no code space.